tests/testthat/test-integrate_sces.R

```diff
@@ -0,0 +1,214 @@
+# generate testing data
+set.seed(1665)
+merged_sce <- sim_sce(n_cells = 300, n_genes = 100, n_empty = 0)
+batches <- c(rep("a", 100),
+             rep("b", 100),
+             rep("c", 100))
+barcodes <- rownames(colData(merged_sce))
+
+# set up colData rownames as batch-barcode
+new_rownames <- tibble::tibble(
+  batches = batches,
+  barcodes = barcodes,
+  ids = glue::glue("{batches}-{barcodes}")) %>%
+  dplyr::pull(ids)
```

Comment:
No need to make the tibble if you aren't saving it
```suggestion
new_rownames <- glue::glue("{batches}-{barcodes}")) 
```